File: exercise/multiplication_table.py
#! /usr/bin/python3
# author: yuankun.wang
# date: 2021/01/02
# function: This grogram is to generate multiplication table.


j = 1
while j < 10:
    i = 1
    while i <= j:
        print("%d*%d=%d" % (i,j,i*j), end=" ")
        i += 1
    j += 1
    print()

File: C/program_example/instances/multiplication_table.c
// /usr/bin/gcc

// File Name       : multiplication_table.c
// Author          : Yuankun.Wang
// Created Time    : 2025-07-06 17:49:03
// Last Modified   : 2025-07-06 17:54:17
#include <stdio.h>

int main() {
    int i, j;
    for (i=1; i<=9; i++) {
        for (j=1; j<=i; j++) {
            printf("%d x %d = %d\t", i, j, i*j);
        }
        printf("\n");
    }
    return 0;
}
